report.qmd

```diff
@@ -1,65 +1,1346 @@
 ---
-title: "Title of your report"
-subtitle: "Project report of Group #"
+title: "Urban stream restoration – Teplica (Senica, SK)"
+subtitle: "Project report of Group C"
 author:
-  - name: "Student 1"
-    affiliation: "MSc Geomatics"
-  - name: "Student 2"
-    affiliation: "MSc Landscape Architecture"
-  - name: "Student 3"
+  - name: "Hongyue Kang"
     affiliation: "MSc Urbanism"
-  - name: "Student 4"
+  - name: "Kelly Schoonderwoerd"
+    affiliation: "MSc Geomatics"
+  - name: "Viola Ebermannová"
     affiliation: "MSc Urbanism"
-  - name: "Student 5"
-    affiliation: "MSc Landscape Architecture"
-format: html
+# format: html
+
+# this toc is not fully fuctional, but at least sth i guess. we might modify the levels still:
+
+format:
+  html:
+    toc: true
+    toc-location: right
+    toc-depth: 5
+    toc-expand: 2
```

Comment:
```suggestion
    toc: true
```
I suggest you dorp the last three lines to allow the report to render properly. There is indeed an issue with the ToC when using these settings.